src/components/BulkEnrollmentPage/stepper/BulkEnrollmentSubmit.jsx

```diff
@@ -126,20 +132,22 @@ const BulkEnrollmentSubmit = ({
         error={error}
         enterpriseId={enterpriseId}
       />
-      <Form.Checkbox
-        checked={checked}
-        onChange={handleChange}
-        data-testid={NOTIFY_CHECKBOX_TEST_ID}
-      >
-        Notify learners via Email
-      </Form.Checkbox>
-      <Button
-        disabled={!hasSelectedCoursesAndEmails && !loading}
-        onClick={submitBulkEnrollment}
-        data-testid={FINAL_BUTTON_TEST_ID}
-      >
-        {FINAL_BUTTON_TEXT}
-      </Button>
+      <Stack direction="horizontal" gap={4}>
```

Comment:
Adds additional space between checkbox and button so they are not right next to each other, and lowercases "Email"